doc/lua_api.txt

```diff
@@ -3663,7 +3663,10 @@ Call these functions only at load time!
     * Return `true` to mark the message as handled, which means that it will
       not be sent to other players.
 * `minetest.register_on_player_receive_fields(func(player, formname, fields))`
-    * Called when a button is pressed in player's inventory form
+    * Called when a button is pressed in player's inventory form, or when the
+      form is closed by the player's keypress
```

Comment:
Or double-click outside the window.
Perhaps: `Called when the player submits form values, presses a button or actively closes the formspec.`